pkg/controller/virtualmachineimport/virtualmachineimport_controller.go

```diff
@@ -365,9 +365,14 @@ func (r *ReconcileVirtualMachineImport) Reconcile(request reconcile.Request) (re
 	}
 
 	if shouldStartVM(instance) {
-		if err = r.startVM(provider, instance, vmName); err != nil {
+		var requeue bool
+		if requeue, err = r.startVM(provider, instance, vmName); err != nil {
 			return reconcile.Result{}, err
 		}
+		if requeue {
+			// No reconciles when vm was updated to run we need to requeue
```

Comment:
I find wording of this comment confusing; could you, please change it? Is it "No reconciles when vm was updated to 'running'; we need to requeue"?

Reply:
Will improve, thanks!